debezium-connector-mysql/src/test/docker/server-gtids/5.5/my.cnf

```diff
@@ -0,0 +1,52 @@
+# For advice on how to change settings please see
```

Comment:
It seems to me that we now have too many files, how about combine two advanatges

* Docker Maven plugin seems to be able to read files from any directory
* the Docker image seems to support `/etc/my.cnf.d/` directory

If this is the case then I propose to create one that is common to all images - this will be copied over `/etc.my.cnf`
Then there will be fragment files containin only Debezium specific settings - those would be copied into `/etc/my.cnf.d` directory to be picked up upon startup.

Reply:
As mentioned on the weekly call, separating the config files introduced test failures however the MySQL documentation covered a configuration pattern option-groups that works which I decided to use instead.